samples/widgets/show-record-id/README.md

```diff
@@ -9,19 +9,21 @@ Clone the [sample repo](https://github.com/esri/arcgis-experience-builder-sdk-re
 Define the data action in the folder of `src/data-actions`. Then extend the base class `AbstractDataAction` and implement its methods: `isSupported` and `onExecute`.
 
 Register the data action in manifest.
-```
+```json
```

Comment:
did we also need a blank line before the code block too?

Reply:
good catch and i added them